src/components/Courses/CourseNameBox.tsx

```diff
@@ -0,0 +1,13 @@
+import { Box, Text, Divider } from "@chakra-ui/react";
+
+export const CourseNameBox = ({ courseCode, courseName, ...other }) => {
+    return (
+        <Box {...other}>
```

Comment:
`codeAndName` is included in `other` so react is complaining
explicitly unpack it like `courseCode` and `courseName`